tests/backup-all-ok.rec

```diff
@@ -72,7 +72,7 @@ mysql -h0 -P9306 -e "flush ramchunk test; select * from test.@files;"
 |   10 | /var/lib/manticore/test/test.0.spds   | /var/lib/manticore/test/test.0.spds   |     64 |
 |   11 | /var/lib/manticore/test/test.0.spknn  | /var/lib/manticore/test/test.0.spknn  |    456 |
 |   12 | /var/lib/manticore/test/test.0.spt    | /var/lib/manticore/test/test.0.spt    |     41 |
-|   13 | /var/lib/manticore/test/test.0.spidx  | /var/lib/manticore/test/test.0.spidx  |    309 |
+|   13 | /var/lib/manticore/test/test.0.spidx  | /var/lib/manticore/test/test.0.spidx  |    149 |
```

Comment:
Should we store digits here? Maybe just use regex?

Reply:
done